repos/fdbt-site/src/pages/ticketConfirmation.tsx

```diff
@@ -237,6 +238,69 @@ export const buildPointToPointPeriodConfirmationElements = (ctx: NextPageContext
     return confirmationElements;
 };
 
+export const buildFlatFarePriceByDistanceConfirmationElements = (
+    ctx: NextPageContextWithSession,
+): ConfirmationElement[] => {
+    const confirmationElements: ConfirmationElement[] = [];
+    const serviceInformation = getSessionAttribute(ctx.req, SERVICE_LIST_ATTRIBUTE) as ServiceListAttribute;
+    const services = serviceInformation ? serviceInformation.selectedServices : [];
+    const pricePerDistance = getSessionAttribute(ctx.req, PRICING_PER_DISTANCE_ATTRIBUTE);
+    const operatorAttribute = getSessionAttribute(ctx.req, OPERATOR_ATTRIBUTE);
+    const opName = operatorAttribute?.name ? `${operatorAttribute.name} ` : '';
+    const dataSource = (getSessionAttribute(ctx.req, TXC_SOURCE_ATTRIBUTE) as TxcSourceAttribute).source;
+    confirmationElements.push(
+        {
+            name: `${opName}${opName ? 's' : 'S'}ervices`,
```

Comment:
Once the above changes, we can assume we have opName.

Reply:
see comment